### PR TITLE
doc/rbd: refine iscsi-initiator-linux.rbd

### PR DESCRIPTION
**Pull request by zdover23**
Add prompts and clean up the lists in doc/rbd/iscsi-initiator-linux.rbd.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
@anthonyeleven @colemitchell: let @idryomov have final approval of this one.

File: doc/rbd/iscsi-initiator-linux.rst [hunk 1]
```diff
@@ -74,40 +74,46 @@ Install the iSCSI initiator and multipath tools:
...
+       mpathbt (360014059ca317516a69465c883a29603) dm-1 LIO-ORG ,IBLOCK
+       size=1.0G features='0' hwhandler='1 alua' wp=rw
+       |-+- policy='queue-length 0' prio=50 status=active
+       | `- 28:0:0:1 sde  8:64  active ready running
+       `-+- policy='queue-length 0' prio=10 status=enabled
+         `- 29:0:0:1 sdc  8:32  active ready running
 
-   ::
+   You should now be able to use the RBD image like you would a normal
+   multipath’d iSCSI disk.
```
Comment (anthonyeleven) [inline doc/rbd/iscsi-initiator-linux.rst:113]: s/'d//

**Review by anthonyeleven — APPROVED** [review]
non-blocking comment

**Comment by zdover23** [comment]
@idryomov: Check this one out, when you have time.

File: doc/rbd/iscsi-initiator-linux.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/rbd/iscsi-initiator-linux.rst:113]: kewl

**Comment by idryomov** [comment]
> English-language improvements will be handled in a future PR.

As discussed earlier, let's do one PR per `.rst` file to minimize the amount of churn.  Leaving stuff on the table when lines in question are actually getting changed (i.e. show up in the diff) is undesirable.

Alternatively, a big PR for just the `.. prompt::` thing covering all of RBD documentation would be fine too.

**Comment by anthonyeleven** [comment]
> > English-language improvements will be handled in a future PR.
> 
> As discussed earlier, let's do one PR per `.rst` file to minimize the amount of churn. Leaving stuff on the table when lines in question are actually getting changed (i.e. show up in the diff) is undesirable.
> 
> Alternatively, a big PR for just the `.. prompt::` thing covering all of RBD documentation would be fine too.

Might be a question of whether there are other updaters for the files in question.  When I harmonized option names to use underscores that would have been a large number of PRs.

**Comment by zdover23** [comment]
> > English-language improvements will be handled in a future PR.
> 
> As discussed earlier, let's do one PR per `.rst` file to minimize the amount of churn. Leaving stuff on the table when lines in question are actually getting changed (i.e. show up in the diff) is undesirable.
> 
> Alternatively, a big PR for just the `.. prompt::` thing covering all of RBD documentation would be fine too.

@idryomov: I'll alter this PR to be a general iscsi-initiator-linux.rbd improvement PR.

**Comment by idryomov** [comment]
> Might be a question of whether there are other updaters for the files in question. When I harmonized option names to use underscores that would have been a large number of PRs.

Harmonizing option names is another example of a bulk near-mechanical change for which a component-wide (or even tree-wide) PR is appropriate.  But note that it's orthogonal to the `.. prompt::` thing -- submitting a separate PR for each such bulk change is fine and even preferred.

File: doc/rbd/iscsi-initiator-linux.rst [hunk 2]
```diff
@@ -74,40 +74,46 @@ Install the iSCSI initiator and multipath tools:
...
+      iscsiadm -m node -T iqn.2003-01.org.linux-iscsi.rheln1 -l
 
 **Multipath IO Setup:**
 
-The multipath daemon (``multipathd``), will set up devices automatically
-based on the ``multipath.conf`` settings. Running the ``multipath``
-command show devices setup in a failover configuration with a priority
-group for each path.
+#. The multipath daemon (``multipathd``) uses the ``multipath.conf`` settings
+   to set up devices automatically. Running the ``multipath`` command shows the
```
Comment (zdover23) [inline doc/rbd/iscsi-initiator-linux.rst:95]: @idryomov: Could you give me more detail on what "Running the ``multipath`` command show devices setup in a failover configuration with a priority group for each path" is trying to express? 

It'd be helpful to have an explanation of what "priority group" means.

Comment (idryomov) [reply doc/rbd/iscsi-initiator-linux.rst:95]: I think it's just trying to say that each path is placed into its own priority group.  Here is a good overview: https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-multipath.html#sec-multipath-grouping

**Comment by zdover23** [comment]
@idryomov: It's ready for your review.

**Review by idryomov — APPROVED** [review]
Please squash everything into a single commit.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49749 - Quincy backport
https://github.com/ceph/ceph/pull/49750 - Pacific backport

**Pull request merged**